sync3/handler/connstate.go

```diff
@@ -96,6 +96,10 @@ func (s *ConnState) load(ctx context.Context) error {
 		rooms[i] = sync3.RoomConnMetadata{
 			RoomMetadata: *metadata,
 			UserRoomData: urd,
+			// Best-effort only: we're not going to scan the database for all events in
+			// the entire room's history to give you a fully accurate timestamp
+			// according to your bump_event_types.
+			LastActivityTimestamp: metadata.LastMessageTimestamp,
```

Comment:
Maybe we should! See https://github.com/vector-im/element-x-ios/issues/413#issuecomment-1559836781